dapr/clients/grpc/_response.py

```diff
@@ -66,7 +66,7 @@ def get_headers(self, as_dict: bool = False) -> Union[MetadataDict, MetadataTupl
 
 
 class InvokeServiceResponse(DaprResponse):
```

Comment:
@yaron2 Should this be renamed to InvokeMethodResponse? and similar changes for other classes?

Reply:
I actually did rename to InvokeMethodResponse/Request, only to realize while testing that we have defined them as InvokeService* in Dapr runtime https://github.com/dapr/java-sdk/blob/6a44d51279d7d6d5ccdfd63346b4c83d00e7eeb8/sdk/src/main/java/io/dapr/client/DaprClient.java, so ended up changing it back.

Jave SDK has also similar naming convention right now https://github.com/dapr/java-sdk/blob/9dcff43ef08497b7169b93207002f450a76236bd/sdk/src/main/java/io/dapr/client/domain/InvokeServiceRequest.java

I do think though that we should rename everything including runtime to InvokeMethod* for consistency, but not sure if it should be prioritized over other things.

Reply:
I noticed that Artur updated Java SDK now to use InvokeMethodResponse/Request, so I've updated the PR to reflect that in python SDK as well. Calls to runtime are still InvokeServiceResponse as I mentioned in the previous comment, just a heads up.